<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>776</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Word-savvy.qrc">
    <normaloff>:/images/Word-savvy.ico</normaloff>:/images/Word-savvy.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="English_tab">
       <attribute name="title">
        <string>English</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="English_tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideRight</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>NUMBER</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>WORD</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PRONUNCIATION</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>MEANING</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>STAR</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>DATE</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>MEMO</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>NUMBER :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="English_number_lineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>WORD : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="English_word_lineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>PRONUNCIATION : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="English_pronunciation_lineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>STAR : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="English_comboBox">
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>**</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>***</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>****</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*****</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>MEANING : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="English_meaning_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>MEMO : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="English_textEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="English_add_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="English_reset_pushButton">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="English_close_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="English_edit_pushButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="English_delete_pushButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="English_checkBox">
              <property name="text">
               <string>Tick</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Keyword : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="keyword_lineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="word_search_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>WORD Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="memo_searchpushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>MEMO Search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Japanese_tab">
       <attribute name="title">
        <string>Japanese</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="Japanese_tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>NUMBER</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>WORD</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PRONUNCIATION</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>MEANING</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>STAR</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>DATE</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>MEMO</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>NUMBER : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Japanese_number_lineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>WORD : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Japanese_word_lineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>PRONUNCIATION : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Japanese_pronunciation_lineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>STAR : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Japanese_comboBox">
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>**</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>***</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>****</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>*****</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>MEANING : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Japanese_meaning_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>MEMO : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="Japanese_textEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="Japanese_add_pushButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Japanese_reset_pushButton">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Japanese_close_pushButton">
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Japanese_edit_pushButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Japanese_delete_pushButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="Japanese_checkBox">
              <property name="text">
               <string>Tick</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionFont"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Word-savvy.qrc"/>
 </resources>
 <connections/>
</ui>
